src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,43 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @Test
+    void split() {
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).containsExactly("1", "2");
```
이와 같이 간단한 테스트의 경우 한 라인으로 구현하는 것은 어떨까?